src/resources/i18n-target.html

```diff
@@ -0,0 +1,52 @@
+<template>
+
+  <button id="locale-changer-de" click.delegate="changeLocale('de')">DE</button>
+
+  <span id="i18n-simple" t="simple.text;[title]simple.attr"></span>
+
+  <span id="i18n-ctx" t="status"></span>
+  <span id="i18n-ctx-dispatched" t="status" t-params.bind="{context: 'dispatched', date: dispatchedOn}"></span>
+  <span id="i18n-ctx-delivered" t="status" t-params.bind="{context: 'delivered', date: deliveredOn}"></span>
+
+  <span id="i18n-items-0" t="itemWithCount" t-params.bind="{count: 0}"></span>
+  <span id="i18n-items-1" t="itemWithCount" t-params.bind="{count: 1}"></span>
+  <span id="i18n-items-10" t="itemWithCount" t-params.bind="{count: 10}"></span>
+
+  <span id="i18n-interval-0" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 0}"></span>
+  <span id="i18n-interval-1" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 1}"></span>
+  <span id="i18n-interval-2" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 2}"></span>
+  <span id="i18n-interval-3" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 3}"></span>
+  <span id="i18n-interval-6" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 6}"></span>
+  <span id="i18n-interval-7" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 7}"></span>
+  <span id="i18n-interval-10" t="itemWithCount_interval" t-params.bind="{postProcess: 'interval', count: 10}"></span>
+
+  <span id="i18n-html" t="[html]html"></span>

```

Comment:
two tests for images could be usesful

1. default translation targets the src attribute
2. data-src to be used as original url

(See docs "Translating Images" https://aurelia.io/docs/plugins/i18n#using-the-plugin)

Reply:
Done for the default `src` translation. Skipped the `data-src` part as it does not get any special treatment.